dashboard/app/mail_bug.txt

```diff
@@ -6,7 +6,7 @@ syzbot {{if .First}}found{{else}}has found a reproducer for{{end}} the following
 
 HEAD commit:    {{formatTagHash .KernelCommit}} {{formatCommitTableTitle .KernelCommitTitle}}
 git tree:       {{.KernelRepoAlias}}
-{{if .LogLink}}console output: {{.LogLink}}
+{{if .LogLink}}{{if .LogHasStrace}}console+strace{{else}}console output{{end}}: {{.LogLink}}
```

Comment:
Need at least 1 test that checks this new output.
Otherwise we won't know if the output changes accidentally, or it the template is broken and crashes.